app/settings/environments.js

```diff
@@ -0,0 +1,25 @@
+import {filter} from 'wasmuth'
+
+const environments = {
+  development: ['http://localhost:3333', 'http://localhost:3334'],
+  staging: '[STAGING_URL]',
+  production: '[PRODUCTION_URL]'
+}
+
+export const environment = (() => {
+  const {host, protocol} = window.location
+  const url = `${protocol}//${host}`
```

Comment:
Do we need to check protocol? Will we have environments that differ between http and https?
Would checking host be enough?

Reply:
yeah good call